fda-model/metrics/Results.cs

```diff
@@ -1,15 +1,18 @@
 using System;
 using System.Collections.Generic;
+using Statistics;
+using Statistics.Histograms;
 namespace metrics
 {
     public class Results: IContainResults
     {
         //needs access to get AEP and EAD results.
+        private const double AEP_HISTOGRAM_BINWIDTH = .01;
+        private const double EAD_HISTOGRAM_BINWIDTH = 10;
```

Comment:
i like both of these, both are reasonable - should this be configurable? like, i only want a histogram accurate to the $1000.00